pkg/querier/querier.go

```diff
@@ -253,6 +256,8 @@ func NewQueryable(distributor QueryableWithFilter, stores []QueryableWithFilter,
 }
 
 type querier struct {
+	cfg Config
```

Comment:
I would prefer to avoid depending on entire Config struct, and rather be explicit about what querier actually uses (`MaxQueryIntoFuture` only for now).